src/constants.js

```diff
@@ -18,5 +18,36 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+// SAMPLED_MASK represents the power of two that indicates we have set the sampled
```

Comment:
a common term is "bitmap mask". And explanation shouldn't refer back to "flag", as it makes is a tautology.

SAMPLED_MASK is a bitmap mask indicating that the trace has been sampled for collection into storage.